src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -1,55 +1,18 @@
 ---
-title: General Configuration
+title: Configuration
 sidebar_order: 410
-supported:
-  - javascript
-  - javascript.angular
-  - javascript.ember
-  - javascript.gatsby
-  - javascript.nextjs
-  - javascript.react
-  - javascript.remix
-  - javascript.svelte
-  - javascript.vue
 notSupported:
   - javascript.capacitor
   - javascript.cordova
   - javascript.electron
   - javascript.wasm
 description: "Learn about the general Session Replay configuration fields."
+redirect_from:
+  - /
```

Comment:
This looks funny, but also the urls are so new do we even need it?

Reply:
Oh I made a mistake here too. Yeah no need to add them